roles/host-provision/tasks/config-tasks.yml

```diff
@@ -17,10 +17,46 @@
   ping:
   tags: host-provision
 
-# Placeholder for the core config tasks to be executed by the host-provision.sh utility
-# - name: config-tasks | Create LVM layer
-# ...
 
+# Not including ntp set up, as we already have code path built into `base-provision` role
```

Comment:
I'm not sure if this comment is even necessary in this context;  it seems self-evident that NTP code already exists.